src/core/model_config.proto

```diff
@@ -579,6 +579,14 @@ message ModelDynamicBatching
   //@@     batching. Default is 0.
   //@@
   uint64 max_queue_delay_microseconds = 2;
+
+  //@@  .. cpp:var:: bool preserve_ordering
+  //@@
+  //@@     If true, the responses will be returned in the same order as the
```

Comment:
Should the dynamic batcher preserve the ordering of responses to match the order of requests received by the scheduler. Default is false. If true, ...